packages/snap-preact/components/src/components/Molecules/Result/Result.tsx

```diff
@@ -219,6 +286,12 @@ export const Result = observer((properties: ResultProps): JSX.Element => {
 						</div>
 					)}
 					{cloneWithProps(detailSlot, { result, treePath })}
+
+					{!hideATCButton && (
+						<div className="ss__result__atc-wrapper">
```

Comment:
Button uses `ss__result__button--addToCart` but this uses `atc` we should be consistent.